bin/oeem-upload

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
```

Comment:
Nice. I assume this is to make it easier to use the uploader during development?